ruby_challenges.rb

```diff
@@ -1,27 +1,71 @@
 # ASSESSMENT 5: Ruby Coding Practical Questions
-
+# MINASWAN :]
 # --------------------1) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data about each bike object.
 
-
-
-
+class Bike
+  attr_accessor :model, :wheels, :frame_size, :bell, :speed
+  def initialize
+  @model = 'Trek'
+  @wheels = 2
+  @frame_size = '168cm'
+  @bell = 'Ding Ding!'
+  @speed = 0
+  end
 
 # Expected output example: 'The Trek bike has 2 wheels and a 168cm frame.'
-
-
+def get_info
+  puts "The #{model} bike has #{wheels} wheels and a #{frame_size} frame."
```

Comment:
Cool use of class methods. Consider why and when you'd want to use getter methods versus instance variables.